components/camel-micrometer/src/main/java/org/apache/camel/component/micrometer/eventnotifier/MicrometerEventNotifierMBean.java

```diff
@@ -14,22 +14,16 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
```

Comment:
Why are you moving a file of camel-metrics for a new component?

Reply:
* Move must have happened accidentally... I restored the original file
* Maybe http://camel.apache.org/how-do-i-add-a-component.html needs an update for occasional Camel contributors like me?
* umm, I did run -Psourcecheck before. Do you see any code style issues?

Reply:
The documentation is old, we are moving to a new website and we are maintaining docs directly on github to be able to be consistent.